horoscope/horoscope.py

```diff
@@ -274,15 +245,18 @@ async def _font(self, ctx, url: str = None):
         await ctx.send("Font has been saved")
```

Comment:
...well that was some dumb code. I never let it download and save the custom font 😅 It would be telling you "yes your font is saved :D ...not."